backend/danswer/connectors/gong/connector.py

```diff
@@ -280,9 +289,12 @@ def poll_source(
         # so adding a 1 day buffer and fetching by default till current time
         start_one_day_offset = start_datetime - timedelta(days=1)
         start_time = start_one_day_offset.isoformat()
+
+        # set end_time to `None` if in the `end` is <= 24 hours ago (e.g.
+        # fetch all available calls). Otherwise, respect the passed in `end` time
         end_time = (
             datetime.fromtimestamp(end, tz=timezone.utc).isoformat()
-            if self.use_end_time
+            if end_datetime < datetime.now(tz=timezone.utc) - timedelta(days=1)
```

Comment:
I think we don't want this right? We want the latest calls if they're processed. If not then... whatever. I can't remember why I even didn't include end time to begin with

Reply:
just in case 🤷

Reply:
but we can also remove it and just always use the end time - also fine with me